index.html

```diff
@@ -51,7 +51,206 @@ <h1 class="first">I'm Recillah Glad To See You!</h1>
                 </li>
             </ul>
         </section> 
-         
-    
+        <!-- Middle section -->
+        <section class="works">
+            <div class="single-card">
+                <div class="card-works">
+                    <a href="#">
+                        <img class="cards-pic" src="Images/Snapshoot-Portfolio-1.png" alt="Snapshoot Portfolio 1">
+                    </a>
+                </div>
+                <div class="left-block">
+                    <div class="primary-text">
+                        <div>
+                            <h2 class="project-title">Tonic</h2>
+                        </div>
+                        <div class="frame-2">
+                            <a class="client" href="#"> CANOPY</a>
+                            <img class="counter" src="Images/Counter.png" alt="dot\fulstop">
+                            <h3 class="role">Back End Dev</h3>
+                            <img class="counter" src="Images/Counter.png" alt="dot\fullstop">
+                            <h4 class="year-label">2015</h4>
+                        </div>
+                        <p class="primary-text">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </p>
+                        <ul class="tags">
+                            <li class="tag">html</li>
+                            <li class="tag">css</li>
+                            <li class="tag">javascript</li>
+                        </ul>
+                        <button class="action">See Project</button>
```

Comment:
- [ ] According to the requirements, You should Implement the button interactions (enable, hover, etc.). Kindly, add the button interactions based on your Figma Design 👌.

![1](https://user-images.githubusercontent.com/45231276/176833382-1d1d826d-6c66-47ac-bec8-36427809918f.PNG)

![image](https://user-images.githubusercontent.com/45231276/177019437-d4c20bc0-7e61-4d62-ac96-2b0d099054c1.png)

- [ ] Note: you should do this on all buttons 👌